### PR TITLE
Added flag for validation validate js

### PR DESCRIPTION
**Pull request by sharonco-matrix**
Brady, we fixed the validation bugs according your comments, please let us know if its acceptable or there is something else to do

### CODE REVIEW

File: interface/forms/newpatient/common.php [hunk 1]
```diff
@@ -65,7 +65,10 @@ function sensitivity_compare($a, $b) {
 <script type="text/javascript" src="<?php echo $GLOBALS['webroot'] ?>/library/textformat.js"></script>
 
 <!-- validation library -->
-<?php  require_once($GLOBALS['srcdir'] . "/validation/validation_script.js.php"); ?>
+<?php
+//LBF forms use the new validation depending on the global value
+$use_validate_js=$GLOBALS['new_validate'];
```
Comment (bradymiller) [inline interface/forms/newpatient/common.php]: This is not an LBF form so, should be using $use_validate_js = 1; strategy.

**Comment by bradymiller** [comment]
Overall, looks like a great strategy. I only noted 1 minor issue. If you address that issue and let me know this is testing well for you, then I'll bring it into the codebase.
thanks, -brady

**Comment by bradymiller** [comment]
oops, didn't mean to close this. just reopened it.

File: interface/forms/newpatient/common.php (hunk 1 above)
Comment (sharonco-matrix) [inline interface/forms/newpatient/common.php]: Oops you are right :) Fixing it now

**Comment by bradymiller** [comment]
looks good and will bring into the codebase. thanks for the contribution.
-brady

File: library/validation/validate_core.php [hunk 2]
```diff
@@ -71,6 +71,8 @@ function validateUsingPageRules($fileNamePath)
         print '<!---Start of page  form validation-->';
         print '<!--//include new rules of submitme functionallity-->';
         echo("\r\n");
+        //Not lbf forms use the new validation, please make sure you have the corresponding values in the list Page validation
+        $use_validate_js = 1;
         require_once($GLOBALS['srcdir'] . "/validation/validation_script.js.php");
         $new_validate = $GLOBALS['new_validate'] ? 1 : 0;
```
Comment (bradymiller) [inline library/validation/validate_core.php]: @sharonco-matrix ,
I am confused by the line above setting $new_validate. Should this be removed and then use $use_validate_js on  line 103?

Comment (sharonco-matrix) [inline library/validation/validate_core.php]: I fixed it in the branch

**Comment by bradymiller** [comment]
Hi @sharonco-matrix ,

Do you have a link to the commit/branch fix on github?

thanks,
-brady

File: library/validation/validate_core.php (hunk 2 above)
Comment (bradymiller) [inline library/validation/validate_core.php]: Hi @sharonco-matrix ,

Do you have a link to the commit/branch fix on github?

thanks,
-brady

**Pull request merged**